template/build/core/phing/tasks/frontend.xml

```diff
@@ -1,4 +1,10 @@
-<project name="frontend" default="frontend:install">
+<project name="frontend" default="frontend:setup">
+
+  <target name="frontend:setup" description="Uses project.yml hooks to run custom defined commands to setup front end dependencies for frontend:build.">
```

Comment:
@grasmash It is different, is "Uses project.yml hooks to run custom defined commands to setup front end dependencies for frontend:build." confusing?